requirements.txt

```diff
@@ -6,6 +6,12 @@ jsonschema==3.2.0
 lz4==3.0.2
 requests==2.23.0
 networkx==2.5
+six~=1.15.0
+python-dateutil==2.8.2
+frozendict~=2.1.0
+filelock~=3.0.12
```

Comment:
I don't see this being used in the PR. Please remove

Reply:
ok

Reply:
Please remove the dep

Reply:
ok

Reply:
Hi @libretto , is this unused dependency is referring to filelock? if so, I still see the dependency in the requirements.txt